src/etl/NFTHelpers.cpp

```diff
@@ -368,3 +372,30 @@ getNFTData(ripple::TxMeta const& txMeta, ripple::STTx const& sttx)
             return {{}, {}};
     }
 }
+
+std::vector<NFTsData>
+getNFTDataFromObj(
+    std::uint32_t const seq,
+    std::string const& key,
+    std::string const& blob)
+{
+    std::vector<NFTsData> nfts;
+    ripple::STLedgerEntry const sle = ripple::STLedgerEntry(
+        ripple::SerialIter{blob.data(), blob.size()},
+        ripple::uint256::fromVoid(key.data()));
+
+    if (sle.getFieldU16(ripple::sfLedgerEntryType) != ripple::ltNFTOKEN_PAGE)
+        return nfts;
+
+    auto const owner = ripple::AccountID::fromVoid(key.data());
```

Comment:
how does this work? isn't this the key to the nft token page?

Reply:
this is how object IDs work for NFTokenPages. The first 160 bits are the owner's account, to save space